src/components/Popup/Popup.js

```diff
@@ -234,6 +234,10 @@ class Popup extends PureComponent {
 
     const { top, left } = this.state;
 
+    // force re-render if the feature or the coordinate changes.
+    // this is needed to update the popupElement ref
+    const key = feature ? feature.getId() : popupCoordinate.join();
```

Comment:
Did you try to use createRef to get the element. It will do the work without this.

Reply:
Should work as well. However, I need the key to force an update of the component.